cmd/influxd/config.go

```diff
@@ -52,13 +52,8 @@ type Config struct {
 		ReadTimeout Duration `toml:"read-timeout"`
 	} `toml:"api"`
 
+	Graphite     []Graphite `toml:"graphite"`
```

Comment:
I think the reason this was left this way was to line up with the way that TOML does collections. I agree I think it should be plural in the code for readability.